{"name": "Логин с неверным логином и паролем", "status": "passed", "parameters": [{"name": "key", "value": "'email'"}], "start": 1742460885609, "stop": 1742460885736, "uuid": "89ac5883-2573-4083-bfc7-f57c0cd94adc", "historyId": "d2bbedebdd7c81dd58d2160e500f515e", "testCaseId": "40f1be16d719b470d6b33a0dcfa51542", "fullName": "tests.test_login.TestLogin#test_login_invalid_data_401", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-4NMQM7J"}, {"name": "thread", "value": "3120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}